sql/moz-fx-data-shared-prod/mozilla_org_derived/ga_sessions_v2/script.sql

```diff
@@ -0,0 +1,376 @@
+        SPLIT(
+          (
+            SELECT
+              `value`
+            FROM
+              UNNEST(event_params)
+            WHERE
+              key = 'page_location'
+            LIMIT
+              1
+          ).string_value,
+          '?'
+        )[OFFSET(0)] AS page_location,
+        event_timestamp
+      FROM
+        `moz-fx-data-marketing-prod.analytics_313696158.events_2*` a
+      JOIN
+        UNNEST(event_params) e
+      WHERE
+        e.key = 'entrances'
+        AND e.value.int_value = 1
+    ),
+    landing_page_by_session AS (
+      SELECT
+        ga_client_id,
+        ga_session_id,
+        page_location,
+        event_timestamp,
+        ROW_NUMBER() OVER (
+          PARTITION BY
+            ga_client_id,
+            ga_session_id
+          ORDER BY
+            event_timestamp ASC
+        ) AS lp_rnk
+      FROM
+        landing_page_by_session_staging
+      QUALIFY
+        lp_rnk = 1
+    ),
+    install_targets_staging AS (
+      SELECT
+        a.user_pseudo_id AS ga_client_id,
+        CAST(e.value.int_value AS string) AS ga_session_id,
+        event_timestamp,
+        event_name AS install_event_name
+      FROM
+        `moz-fx-data-marketing-prod.analytics_313696158.events_2*` a
+      JOIN
+        UNNEST(event_params) e
+      WHERE
+        e.key = 'ga_session_id'
+        AND e.value.int_value IS NOT NULL
+        AND a.event_name IN (
+          'firefox_download',
+          'firefox_mobile_download',
+          'focus_download',
+          'klar_download'
+        )
+    ),
+    all_install_targets AS (
+      SELECT
+        ga_client_id,
+        ga_session_id,
+        ARRAY_AGG(install_event_name) AS all_reported_install_targets,
+        ARRAY_AGG(install_event_name ORDER BY event_timestamp DESC)[
+          0
+        ] AS last_reported_install_target
+      FROM
+        install_targets_staging
+      GROUP BY
+        ga_client_id,
+        ga_session_id
+    )
+    SELECT
+      a.ga_client_id,
+      a.ga_session_id,
+      a.session_date,
+      CASE
+        WHEN a.ga_session_number = 1
+          THEN TRUE
+        ELSE FALSE
+      END AS is_first_session,
+      a.ga_session_number AS session_number,
+      b.max_event_timestamp - b.min_event_timestamp AS time_on_site,
+      b.pageviews,
+      a.country,
+      a.region,
+      a.city,
+      a.campaign_id,
+      a.campaign,
+      a.source,
+      a.medium,
+      a.term,
+      a.content,
+      a.gclid,
+      a.device_category,
+      a.mobile_device_model,
+      a.mobile_device_string,
+      a.os,
+      a.os_version,
+      a.language,
+      a.browser,
+      a.browser_version,
+      b.had_download_event,
+      g.last_reported_install_target,
+      g.all_reported_install_targets,
+      d.last_reported_stub_session_id,
+      d.all_reported_stub_session_ids,
+      e.page_location AS landing_screen
+    FROM
+      device_properties_at_session_start_event a
+    LEFT JOIN
+      event_aggregates b
```

Comment:
Could make this a bit shorter and easier to read with `USING()` instead of `ON`

Reply:
fixed, thanks!